package.json

```diff
@@ -17,51 +17,57 @@
     "e2e": "TS_NODE_PROJECT=e2e/tsconfig.json cucumber-js --require-module ts-node/register './e2e/features/' --require './e2e/step_definitions/*.ts' --require './e2e/support/*.ts' -f json:e2e/reporting/json/json-report.json --format summary",
     "e2e:local": "SELENIUM_BROWSER=chrome TS_NODE_PROJECT=e2e/tsconfig.json cucumber-js --require-module ts-node/register './e2e/features/' --require './e2e/step_definitions/*.ts' --require './e2e/support/*.ts'",
     "e2e-report": "TS_NODE_PROJECT=e2e/tsconfig.json ts-node e2e/reporting/reporter.ts",
-    "lint": "eslint --ext=ts,tsx src/**/* e2e/support e2e/step_definitions e2e/pages"
+    "lint": "eslint --ext=ts,tsx src/**/* e2e/support e2e/step_definitions e2e/pages",
+    "ncu": "ncu"
   },
   "dependencies": {
     "@apollo/react-hooks": "^3.1.2",
-    "@material-ui/core": "^4.5.0",
-    "@material-ui/icons": "^4.4.3",
+    "@material-ui/core": "^4.7.0",
+    "@material-ui/icons": "^4.5.1",
+    "@material-ui/lab": "^4.0.0-alpha.32",
```

Comment:
Any concerns about this breaking/being unstable?

Reply:
no, that's just a minor release but nothing is braking as far as I can see